clients/js/src/createMint.ts

```diff
@@ -0,0 +1,71 @@
+import { getCreateAccountInstruction } from '@solana-program/system';
+import {
+  Address,
+  InstructionPlan,
+  OptionOrNullable,
+  sequentialInstructionPlan,
+  TransactionSigner,
+} from '@solana/kit';
+import {
+  getInitializeMint2Instruction,
+  getMintSize,
+  TOKEN_PROGRAM_ADDRESS,
+} from './generated';
+
+// RPC `getMinimumBalanceForRentExemption` for 82 bytes
+// Hardcoded to avoid requiring an RPC request each time
+const minimumBalanceForMint = 1461600;
+
+export type GetOrCreateMintInstructionPlanInput = {
```

Comment:
I'm guessing here (and on the config type below) you meant `GetCreate...` instead of `GetOrCreate...`. But I think it might make more sense to make them start directly with `Create...`. I believe this is the convention we use for generated helpers.